features/step_definitions/session_steps.rb

```diff
@@ -66,6 +66,12 @@
   end
 end
 
+When "{buyer} logs in" do |buyer|
```

Comment:
```suggestion
When "buyer {buyer} logs in" do |buyer|
```

it doesn't look so nice in the step definition. But in the scenario it would be

```
When buyer "Steven" logs in
```

Without the `buyer` it could as well be a provider when one reads the scenario.